.github/workflows/diagram-cache.yml

```diff
@@ -0,0 +1,54 @@
+# Copyright DB Netz AG and contributors
+# SPDX-License-Identifier: Apache-2.0
+
+name: update_capella_diagram_cache
+
+on: push
+
+env:
+  CAPELLA_DOCKER_IMAGES_REVISION: main
+
+  # Can either be no or selected
+  CAPELLA_DOCKER_IMAGES_DROPINS: selected
+
+jobs:
+  update_capella_diagram_cache:
+      runs-on: ubuntu-latest
+      steps:
+        - name: Checkout code
+          uses: actions/checkout@v3
+        - name: Set up Python 3.11
+          uses: actions/setup-python@v3
+          with:
+            python-version: 3.11
+        - name: Install capellambse
+          run:
+            pip install "capellambse[cli]@git+https://github.com/DSD-DBS/py-capellambse.git@$CAPELLAMBSE_REVISION"
+        - name: Create diagram cache
+          run: |
+            python << EOF
+            import capellambse
+            import os
+            model = capellambse.MelodyModel(
+              os.environ["ENTRYPOINT"], diagram_cache="./diagram_cache"
+            )
+            model.update_diagram_cache(
+                "ghcr.io/dsd-dbs/capella-dockerimages/capella/base:{VERSION}-${{ env.CAPELLA_DOCKER_IMAGES_DROPINS }}-dropins-${{ env.CAPELLA_DOCKER_IMAGES_REVISION }}",
+                create_index=True,
+            )
+            EOF
```

Comment:
This comment doesn't seem to applied yet: https://github.com/DSD-DBS/coffee-machine/pull/4#discussion_r1245415006

Reply:
good catch - apparently it got lost on the way...